x-pack/plugins/monitoring/server/routes/api/v1/apm/overview.ts

```diff
@@ -43,7 +43,9 @@ export function apmOverviewRoute(server: MonitoringCore) {
       try {
         const [stats, metrics] = await Promise.all([
           getApmClusterStatus(req, { clusterUuid }),
-          getMetrics(req, 'beats', metricSet),
+          getMetrics(req, 'beats', metricSet, [
+            { term: { 'beats_stats.beat.type': 'apm-server' } },
```

Comment:
We were computing metrics from **all** beats instances in the apm overview since we didn't filter for apm-server